specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/sqlServer.json

```diff
@@ -1526,7 +1526,11 @@
         },
         "isStorageSecondaryKeyInUse": {
           "description": "Specifies whether storageAccountAccessKey value is the storage's secondary key.",
-          "type": "boolean"
+          "type": "boolean",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
```

Comment:
Can you explain the intent and purpose of this change? You've omitted "read" from the mutability list -- does that mean that is property can be updated but will not be returned?

Reply:
I made changes as suggested in S360

Reply:
I'll update it to create, read